datasafe-rest-impl/src/main/java/de/adorsys/datasafe/rest/impl/controller/VersionController.java

```diff
@@ -107,6 +140,11 @@ public void deleteVersionedDocument(@RequestHeader String user,
      * list of file versions.
      */
     @GetMapping(value = "/versions/list/{path:.*}", produces = APPLICATION_JSON_VALUE)
+    @ApiOperation("List versions of document")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "List command successfully completed"),
+            @ApiResponse(code = 403, message = "Access denied")
```

Comment:
403 has default message, not necessary to add it

Reply:
removed